tests/test_partitioning_time.py

```diff
@@ -333,6 +383,45 @@ def test_partitioning_time_daily_apply_insert():
     model.objects.create(timestamp=datetime.date(2019, 1, 10))
 
 
+@pytest.mark.postgres_version(lt=110000)
```

Comment:
Is this Postgres 11? We use 14, so let's use the closest to ours.

Reply:
Or use both if possible.

Reply:
This decorator only specifies minimum version of PG required for this test to be executed. Actual version of PG I've used is the one in docker-compose (`postgis/postgis:14-3.2`). however this feature should work for any version of PG greater or equal 11,  thus this decorator is set to `lt=110000`. 
In PR to public repo I'll make additional tests on PG 11, as at the moment I've focused only on our use case.